docs/concepts/cluster-administration/sysctl-cluster.md

```diff
@@ -130,19 +130,21 @@ to schedule those pods onto the right nodes.
 
 ## PodSecurityPolicy Annotations
 
-The use of sysctl in pods can be controlled via annotations on the PodSecurityPolicy.
+The use of sysctl in pods can be controlled via annotation on the PodSecurityPolicy.
 
-Here is an example, it authorizes binding user creating pod with corresponding
-_safe_ and _unsafe_ sysctls.
+Sysctl annotations represents the key of a whitelist of allowed safe and unsafe sysctls
```

Comment:
s/annotations/annotation/

s/represents the key of a whitelist of/represents a whitelist of/ ?